charts/nginx-gateway-fabric/values.schema.json

```diff
@@ -519,6 +519,20 @@
           "required": [],
           "title": "securityContext",
           "type": "object"
+        },
+        "snippetsFilters": {
```

Comment:
Where did this come from? This file is autogenerated based on the helm values file.

Reply:
Yes something is off here in terms of generated files. I am investigating but I haven't changed values in values.yaml.

Reply:
Maybe we pushed updates into main but didn't regenerate the feature branch, so this is just a result of that.

Reply:
Hmmm, how do we go about fixing this? This will anyway be merged onto the feature branch so we can regenerate files and see the diff with main, merge as needed?

Reply:
Yeah, if the pipeline is passing here, then I think these generations were necessary and just missed in the rebase.

Reply:
>Maybe we pushed updates into main but didn't regenerate the feature branch, so this is just a result of that. 

I think this is right. The new changes to the Helm validation got pushed to main, but never got pushed to the feature branch. So when we rebased the `feature/snippets-filter` branch onto main, this PR generated this.